com.unity.netcode.adapter.utp/Runtime/NetworkMetricsPipelineStage.cs

```diff
@@ -0,0 +1,53 @@
+#if MULTIPLAYER_TOOLS
+using AOT;
+using Unity.Burst;
```

Comment:
Not strictly required (UTP already depends on it), but if we're using Burst directly in the package, we should list it as a dependency in the `package.json` file.

Reply:
Will do.